.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 3
+charset = utf-8
+end_of_line = crlf
```

Comment:
`end_of_line` should be `lf` for anything edited manually - maybe the counting was thrown off by the fact that the protocol uses `crlf` ?